spec/01-unit/01-db/01-schema/06-routes_spec.lua

```diff
@@ -262,9 +262,9 @@ describe("routes schema", function()
 
         local ok, err = Routes:validate(route)
         assert.falsy(ok)
-        assert.equal(u([[invalid regex: '/users/(foo/profile' (PCRE returned:
+        assert.equal(u([[invalid regex: '/users/(foo|profile' (PCRE returned:
```

Comment:
the set of valid RFC 3986 characters doesn't really affect the regex validation on routes.

I recommend moving the test below to test `services.path` attribute instead